UPGRADE-3.0.md

```diff
@@ -125,13 +253,228 @@ UPGRADE FROM 2.x to 3.0
    closures, but the closure is now resolved in the type instead of in the
    loader.
 
+ * Using the entity provider with a Doctrine repository implementing `UserProviderInterface` is not supported anymore.
+   You should make the repository implement `UserLoaderInterface` instead.
+
 ### EventDispatcher
 
+ * The method `getListenerPriority($eventName, $listener)` has been added to the
+   `EventDispatcherInterface`.
  * The interface `Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcherInterface`
    extends `Symfony\Component\EventDispatcher\EventDispatcherInterface`.
 
 ### Form
 
+ * The `getBlockPrefix()` method was added to the `FormTypeInterface` in replacement of
+   the `getName()` method which has been removed.
+
+ * The `configureOptions()` method was added to the `FormTypeInterface` in replacement
+   of the `setDefaultOptions()` method which has been removed.
+
+ * The `getBlockPrefix()` method was added to the `ResolvedFormTypeInterface` in
+   replacement of the `getName()` method which has been removed.
+
+ * The `cascade_validation` option was removed. Use the `constraints` option
+   together with the `Valid` constraint instead.
+
+ * Type names were removed. Instead of referencing types by name, you must
+   reference them by their fully-qualified class name (FQCN) instead:
```

Comment:
That's wrong. All these Form changes are part of Symfony 2.8, so they have nothing to do in the UPGRADE file in the 2.7 branch